tox.ini

```diff
@@ -1,11 +1,17 @@
 [tox]
 envlist =
     lint
-    py{35,36,37}-django111
-    py{35,36,37,38}-django22
+    py{36,37}-django111
+    py{36,37,38}-django22
     py{36,37,38}-django30
     py{36,37,38}-djangomaster
 
+[gh-actions]
+python =
+    3.6: py36, lint
```

Comment:
This isn't completely analogous to how Travis worked. Travis had an entirely separate matrix build running for the `lint` environment, and this runs the `lint` environment as part of the `3.6` build. If there is a lint error, then only the `3.6` build will fail with a lint error message in the build output.

It's kinda tricky to `include` a separate matrix build here that still uses `3.6` (which is in the matrix) while applying a `TOXENV` variable like we did in `.travis.yml`. Actions is gonna want to just stick the `TOXENV` variable onto the existing `3.6` build. If we really wanted this to be a separate job it might be easier to split linting out into a separate workflow or a conditional job to avoid running it with every version of Python, but that feels like it adds more complexity than it's worth.

Reply:
At first glance, it takes a little while to make sense of how this is working and why, but I think it is a reasonable approach as-is. Alternative approaches to linting against only one version seem like they'd have at least the same number of pitfalls, if not more.